### PR TITLE
refactor: change message interface

### PR DESCRIPTION
**Pull request by luizmuraro**
Change message interface to receive data instead of message.

### CODE REVIEW

**Review by ribeirojose — APPROVED** [review]
👌

File: www/public/mockServiceWorker.js [hunk 1]
```diff
@@ -8,128 +8,128 @@
  * - Please do NOT serve this file on production.
  */
 
-const PACKAGE_VERSION = "2.3.5";
```
Comment (ribeirojose) [inline www/public/mockServiceWorker.js:11]: formatting changes only ⚠️

Comment (luizmuraro) [reply www/public/mockServiceWorker.js:11]: My bad!

File: www/src/components/ui/hooks/use-copy-message.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  const { toast } = useToast();
+
+  const handleCopyMessage = useCallback(
+    (textToCopy: string) => {
+      navigator.clipboard.writeText(textToCopy).then(() => {
+        toast({
+          title: "Copied to clipboard",
+          description: "The message has been copied to your clipboard.",
+        });
+      });
```
Comment (ribeirojose) [inline www/src/components/ui/hooks/use-copy-message.ts:15]: `#would` use async/await, but curious to understand if you have a preference for using callbacks

**Review by ribeirojose — APPROVED** [review]
nice refactors! 🙇

File: www/src/components/ui/hooks/use-copy-message.ts (hunk 2 above)
Comment (luizmuraro) [reply www/src/components/ui/hooks/use-copy-message.ts:15]: Tbh If I'm not mistaken, this function was already like that, then I've just copy and paste to this hook 😬 , but it's fixed and with error handling now!

**Review by ribeirojose — APPROVED** [review]
btw, congrats on the commit messages and organization 🙇

**Pull request merged**